Week-1/index.html

```diff
@@ -0,0 +1,219 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>CattyBoard</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+      body {
+        font-family: "Lato", sans-serif;
+      }
+
+      .sidebar {
+        height: 80%;
+        width: 50px;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: whitesmoke;
+        overflow-x: hidden;
+        padding-top: 16px;
+      }
+
+      .sidebar i {
+        padding: 6px 8px 50px 16px;
+        text-decoration: none;
+        font-size: 20px;
+        color: black;
+        display: block;
+      }
+
+      .head {
+        margin-left: 50px; /* Same as the width of the sidenav */
+        height: 30%;
+        width: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: gray;
+        padding: 0px 0px;
+        overflow-x: hidden;
+      }
+
+      #headImage {
+        padding: 6px;
+        float: left;
+      }
+      #headText {
+        float: left;
+        color: white;
+      }
+
+      #headText i {
+        padding: 16px;
+        margin-top: 50px;
+        margin-left: 0px;
+        text-decoration: none;
+        font-size: 20px;
+      }
+
+      table {
+        margin-left: 50px; /* Same as the width of the sidenav */
+        margin-top: 16%;
+        height: 53%;
+        width: 100%;
+        position: fixed;
+        top: 0;
+        left: 10px;
+        right: 10px;
+        overflow-x: hidden;
+        border-collapse: collapse;
+        border-spacing: 0px;
+      }
+
+      table td {
+        border: none;
+        text-align: left;
+      }
+
+      table td button {
+        color: blue;
+      }
+
+      table tr {
+        height: 0px;
+      }
+
+      table tr:nth-child(even) {
+        background-color: whitesmoke;
+      }
+
+      .foot {
+        margin-left: 0px; /* Same as the width of the sidenav */
+        margin-top: 45%;
+        height: 20%;
+        width: 100%;
+        position: fixed;
+        top: 0px;
+        left: 0px;
+        right: 0px;
+        overflow-x: hidden;
+        background-color: gray;
+      }
+
+      .foot img {
+        float: left;
+      }
+
+      #footText {
+        float: left;
+        padding: 30px;
+        color: white;
+        text-align: center;
+      }
+
+      #footIcons {
+        float: left;
+        margin-left: 250px;
+        margin-block: 60px;
+        color: white;
+        text-align: center;
+      }
+
+      #footIcons i {
+        margin-inline: 60px;
+      }
+
+      #headImage img {
+        height: 80%;
+        width: 80%;
+      }
+
+      .foot img {
+        margin-top: 10px;
+        margin-bottom: 10px;
+        margin-left: 10px;
+        margin-right: 10px;
+        height: 80%;
+        width: 20%;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="sidebar">
```

Comment:
Use Nav, Aside, section tags and also the class name should be in a specific pattern